Test/Integration/Export/ProductExportDataTest.php

```diff
@@ -0,0 +1,105 @@
+    protected $productCollectionFactory;
+
+    protected function setUp(): void
+    {
+        $this->productCollectionFactory = Bootstrap::getObjectManager()->create(\Magento\Catalog\Model\ResourceModel\Product\CollectionFactory::class);
+        $this->productExportManager = Bootstrap::getObjectManager()->create(\StoreKeeper\StoreKeeper\Model\Export\ProductExportManager::class);
+    }
+    /**
+     * @magentoConfigFixture current_store storekeeper_general/general/storekeeper_shop_language nl
+     * @magentoConfigFixture current_store general/locale/code nl_NL
+     * @magentoConfigFixture current_store tax/calculation/price_includes_tax 1
+     * @magentoConfigFixture current_store tax/display/type 3
+     * @magentoConfigFixture current_store tax/defaults/country NL
+     * @magentoDataFixture StoreKeeper_StoreKeeper::Test/Integration/_files/categories.php
+     * @magentoDataFixture StoreKeeper_StoreKeeper::Test/Integration/_files/tax_classes.php
+     * @magentoDataFixture StoreKeeper_StoreKeeper::Test/Integration/_files/taxable_simple_product.php
+     */
+    public function testGetProductExportData()
+    {
+        $this->assertEquals($this->getTestProductExportData(), $this->getProductExportData());
+    }
+
+    /**
+     * @return array
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function getProductExportData(): array
+    {
+        $productCollection = $this->productCollectionFactory->create();
+        $productCollection->addFieldToSelect('*');
+        $productCollection->setStoreId(\Magento\Store\Model\Store::DEFAULT_STORE_ID);
+        $productCollection->addMediaGalleryData();
+        $productCollection->setOrder('entity_id', 'asc');
+        $productExportData = $this->productExportManager->getProductExportData($productCollection->getItems());
+
+        return $this->getFoundEntityData('taxable_product', $productExportData, 'path://product.sku');
+    }
+
+    /**
+     * @return array
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
+     */
+    public function getTestProductExportData(): array
+    {
+        return [
+            'path://product.type' => 'simple',
+            'path://product.sku' => 'taxable_product',
+            'path://title' => 'Taxable Product',
+            'path://summary' => 'Test short description',
+            'path://body' => 'Test description',
+            'path://slug' => 'taxable-product',
+            'path://seo_title' => 'Test meta title',
+            'path://seo_keywords' => 'Test meta keyword',
+            'path://seo_description' => 'Test meta description',
+            'path://product.active' => 'yes',
+            'path://product.product_stock.in_stock' => 'yes',
+            'path://product.product_stock.value' => 100,
+            'path://product.product_stock.unlimited' => 'yes',
+            'path://shop_products.main.backorder_enabled' => 'no',
+            'path://product.product_price.tax' => 21.0,
```

Comment:
```suggestion
            'path://product.product_price.tax' => 0.21,
```
Tax is `0 <= x <= 1`. Did you try to import the generated csv in import export center in storekeeper?

Reply:
@ikulis 21 is not the percent, but tax amount, similarly how it is calculated by Magento. I was not aware that it awaits tax percent/multiplier. About "Import export center" I also was not aware about its existence, it is not stated anywhere in tasks or our conversations. But I will check that